interfaces/cython/cantera/ck2yaml.py

```diff
@@ -2045,6 +2045,10 @@ def convert_mech(input_file, thermo_file=None, transport_file=None,
             phase_name = None
 
         if surface_file:
+            if phase_name is None:
+                msg = "Cannot specify a surface mechanism without a gas phase"
+                logger.warning(f"\nERROR: {msg}\n")
```

Comment:
Should this be `logger.error`?

Reply:
Very good point, but while this was my first intuition also, it is not what it is used elsewhere in this file. I'd rather stick with the current convention and address this at a later point. I added the output that is generated using the current logger to the description on top.

Reply:
@bryanwweber ... let me know how you'd like to resolve this.

Reply:
Sorry, sounds good to me!

Reply:
Great! I think the fix should be overall uncontroversial - it’s ready to merge …